src/web-router/components/connections/index.jsx

```diff
@@ -7,8 +7,16 @@ import Senders from './senders-component'
 import Routes from './routes-component'
 import Receivers from './receivers-component'
 import ChangeMode from './change-mode-component'
+import ScrollWindowing from '../scroll-windowing/scroll-windowing'
+import { ROUTABLE_ROW_HEIGHT } from '../scroll-windowing/constants'
+
+const filterByUnchecked = ({ stateString }) => !stateString.includes('unchecked')
```

Comment:
I think i need a comment here to understand the significance of unchecked

Reply:
and maybe something at this top level that says:

- We filter so that only chosen routables are rendered to the DOM
- We then window so that only those on screen are re-rendered on scroll

Reply:
Comments added - https://github.com/bbc/nmos-web-router/pull/16/commits/052db1f72029c530a1adf554504f1c589234b7b6

Reply:
looks awesome